guacamole/src/main/webapp/app/rest/services/activeConnectionService.js

```diff
@@ -66,68 +65,6 @@ angular.module('rest').factory('activeConnectionService', ['$injector',
 
     };
 
-    /**
-     * Returns a promise which resolves with all active connections accessible
-     * by the current user, as a map of @link{ActiveConnection} maps, as would
-     * be returned by getActiveConnections(), grouped by the identifier of
-     * their corresponding data source. All given data sources are queried. If
-     * an error occurs while retrieving any ActiveConnection map, the promise
-     * will be rejected.
-     *
-     * @param {String[]} dataSources
-     *     The unique identifier of the data sources containing the active
-     *     connections to be retrieved. These identifiers correspond to
-     *     AuthenticationProviders within the Guacamole web application.
-     *
-     * @param {String[]} [permissionTypes]
-     *     The set of permissions to filter with. A user must have one or more
-     *     of these permissions for an active connection to appear in the
-     *     result.  If null, no filtering will be performed. Valid values are
-     *     listed within PermissionSet.ObjectType.
-     *
-     * @returns {Promise.<Object.<String, Object.<String, ActiveConnection>>>}
-     *     A promise which resolves with all active connections available to
-     *     the current user, as a map of ActiveConnection maps, as would be
-     *     returned by getActiveConnections(), grouped by the identifier of
-     *     their corresponding data source.
-     */
-    service.getAllActiveConnections = function getAllActiveConnections(dataSources, permissionTypes) {
```

Comment:
Is this method not used?

Reply:
Not used at all. I'm guessing it must have been added before `dataSourceService` was created, as the only place in the webapp where `getAllActiveConnections()` would have been called is using `dataSourceService` and `getActiveConnections()` instead:

https://github.com/apache/guacamole-client/blob/9a2e0c60879ce0b7dc14870cdb5fe686e2c22eef/guacamole/src/main/webapp/app/settings/directives/guacSettingsSessions.js#L224-L228